courses/management/utils_test.py

```diff
@@ -37,3 +52,128 @@ def test_fetch_enrollment(order):
 
     assert enrolled_obj == program_enrollment.program
     assert enrollment_obj == program_enrollment
+
+
+@pytest.mark.parametrize("command_options", [{"program": "fake", "run": "fake"}, {}])
+def test_fetch_enrollment_parameters(command_options):
+    """Test that fetch_enrollment raises errors on wrong parameters combination"""
+    user = UserFactory()
+    with pytest.raises(CommandError):
+        EnrollmentChangeCommand.fetch_enrollment(user, command_options)
+
+
+def test_fetch_enrollment_inactive():
+    """Test that fetch_enrollment raises an error on inactive enrollment"""
+    enrollment = CourseRunEnrollmentFactory(active=False)
+    with pytest.raises(CommandError):
+        EnrollmentChangeCommand.fetch_enrollment(
+            enrollment.user, {"run": enrollment.run.courseware_id}
+        )
+
+
+def test_create_run_enrollment():
+    """Test that the EnrollmentChangeCommand helper method works properly"""
+    user = UserFactory()
+    user_2 = UserFactory()
+    enrollment = CourseRunEnrollmentFactory(user=user, edx_enrolled=False)
+
+    assert enrollment.edx_enrolled is False
+
+    with patch.object(utils, "enroll_in_edx_course_runs"):
```

Comment:
We have some patching patterns in our test suite that you should apply here. [Here's an example](https://github.com/mitodl/mitxpro/blob/master/courses/api_test.py#L134) that patches this exact function in a different module, and [here's another example](https://github.com/mitodl/mitxpro/blob/master/hubspot/api_test.py#L277) that patches a function with a return value. Please review [this doc](https://docs.python.org/3/library/unittest.mock.html#id6) if you're not clear about the path that you need to provide to `patch()` in order to correctly patch a function. After changing these patches to follow our existing patterns, this test would look more like...

```python
def test_create_run_enrollment(mocker):
  patched_enroll_in_edx = mocker.patch("courses.management.utils. enroll_in_edx_course_runs")
  patched_send_email = mocker.patch("courses.management.mail_api.send_course_run_enrollment_email")
  command = EnrollmentChangeCommand()
  enrollment = command.create_run_enrollment()
  patched_send_email.assert_called_once()
```

Please apply this pattern for all of your tests in this PR, and for any open/future PRs you put up.